src/main/java/core/basesyntax/FigureInfo.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureInfo {
+    String showInfo();

```

Comment:
```suggestion
    String getInfo();

```
Becouse, it returns the info. Otherwise, it should be void